servers/movie_writer/movie_writer.h

```diff
@@ -56,6 +57,8 @@ class MovieWriter : public Object {
 	static uint32_t writer_count;
 
 protected:
+	String base_path;
```

Comment:
I pulled `base_path` out of the `.avi` and `.png` movie writers and into the base `MovieWriter` so I could have access to it for the [logging](https://github.com/godotengine/godot/pull/90025/files#diff-f22fa838d4423c7b6ae578e505e1eb56c2128c1a3a6681c0236b2ff5565cf70dR208)